bus/bus.go

```diff
@@ -944,10 +968,11 @@ func New(s Syncer, cm ChainManager, tp TransactionPool, w Wallet, hdb HostDB, ms
 // Handler returns an HTTP handler that serves the bus API.
 func (b *bus) Handler() http.Handler {
 	return jape.Mux(tracing.TracedRoutes("bus", map[string]jape.Handler{
-		"GET    /accounts/:owner":         b.accountsOwnerHandlerGET,
-		"POST   /accounts/:id/add":        b.accountsAddHandlerPOST,
-		"POST   /accounts/:id/update":     b.accountsUpdateHandlerPOST,
-		"POST   /accounts/:id/resetdrift": b.accountsResetDriftHandlerPOST,
+		"GET    /accounts/:owner":           b.accountsOwnerHandlerGET,
+		"POST   /accounts/:id/add":          b.accountsAddHandlerPOST,
+		"POST   /accounts/:id/update":       b.accountsUpdateHandlerPOST,
```

Comment:
`nit` seems like `/accounts/:id/add` is really more like an update and update should/could be `/accounts/:id/balance`?

Reply:
You think so? I chose `/add` to make it clear that we are not setting a balance but increment it. So it's safe to call from multiple threads because the order doesn't matter. Whereas `/update` changes not only the balance but also the drift to a given value. So the order matters.